pyfolio/plotting.py

```diff
@@ -1734,27 +1728,181 @@ def plot_multistrike_cones(is_returns, oos_returns, num_samples=1000,
+    cone of forecasted cumulative returns. This cone is non-parametric,
+    meaning it does not assume that returns are normally distributed. Redraws
+    a new cone when returns fall outside of last cone drawn.
+    Parameters
+    ----------
+    is_returns : pandas.core.frame.DataFrame
+        Non-cumulative in-sample returns.
+    oos_returns : pandas.core.frame.DataFrame
+        Non-cumulative out-of-sample returns.
+    num_samples : int
+        Number of samples to draw from the in-sample daily returns.
+        Each sample will be an array with length num_days.
+        A higher number of samples will generate a more accurate
+        bootstrap cone.
+    name : str, optional
+        Plot title
+    ax : matplotlib.Axes, optional
+        Axes upon which to plot.
+    cone_std : list of int/float
+        Number of standard devations to use in the boundaries of
+        the cone. If multiple values are passed, cone bounds will
+        be generated for each value.
+    random_seed : int
+        Seed for the pseudorandom number generator used by the pandas
+        sample method.
+    num_strikes : int
+        Upper limit for number of cones drawn. Can be anything from 0 to 3.
+
+    Returns
+    -------
+    Returns are either an ax or fig option, but not both. If a
+    matplotlib.Axes instance is passed in as ax, then it will be modified
+    and returned. This allows for users to plot interactively in jupyter
+    notebook. When no ax object is passed in, a matplotlib.figure instance
+    is generated and returned. This figure can then be used to save
+    the plot as an image without viewing it.
+    ax : matplotlib.Axes
+        The axes that were plotted on.
+    fig : matplotlib.figure
+        The figure instance which contains all the plot elements.
+    """
+    bounds = timeseries.forecast_cone_bootstrap(
+        is_returns=is_returns,
+        num_days=len(oos_returns),
+        cone_std=cone_std,
+        num_samples=num_samples,
+        random_seed=random_seed
+    )
+    return plot_cones(
+        name=name,
+        bounds=bounds,
+        oos_returns=oos_returns,
+        num_samples=num_samples,
+        ax=ax,
+        cone_std=cone_std,
+        random_seed=random_seed,
+        num_strikes=num_strikes
+    )
+
+
+def plot_portfolio_cones(live_all, ret_all, start_dates,
+                         num_samples=1000, ax=None, cone_std=(1., 1.5, 2.),
+                         random_seed=None, num_strikes=3):
+    """
+    Plots the upper and lower bounds of an n standard deviation
+    cone of forecasted cumulative returns. Redraws a new cone when returns
+    fall outside of last cone drawn.
+
+    Parameters
+    ----------
+    live_all : pandas.core.panel.Panel
+        Account names are items, the major axis is
+        pandas.tseries.index.DatetimeIndex, and the minor axis contains a
+        'returns' column. Each 'returns' column contains non-cumulative
+        in-sample returns.
```

Comment:
Looks like this should be out of sample? Since we generate `oos_returns` from `live_all`